glusterapi/bitrot.py

```diff
@@ -0,0 +1,66 @@
+import httplib
+
+from glusterapi.common import BaseAPI
+from glusterapi.exceptions import GlusterApiInvalidInputs
+
+
+class BitrotApis(BaseAPI):
+    def bitrot_enable(self, volume=""):
```

Comment:
Volume need not have default empty string. Since we use it, let bitrot_* functions fail with TypeError if user does not provide any parameter.